packages/cdktf/lib/testing/adapters/__tests__/index.test.ts

```diff
@@ -37,6 +37,9 @@ describe("#synthScope", () => {
   });
 
   test("using resource", () => {
+    /**
+     *
+     */
```

Comment:
could we ignore `*.test.ts` files for the jsdoc rules?
Else I'd rather but an eslint-disable comment at the root of each test and get rid of these empty comments

Reply:
nice!